examples/react-app/package.json

```diff
@@ -16,6 +16,7 @@
     "@tanstack/react-query": "^5.18.1",
     "axios": "^1.6.7",
     "form-data": "~4.0.0",
+    "prettier": "^3.2.5",
```

Comment:
Is this package necessary for this modification? I am considering [Biome](https://biomejs.dev/) as a tool for linting and formatting in terms of performance and ease of implementation.

Reply:
I created the PR #76.

Reply:
This is needed as a test for users of the repo, as described. 
Prettier and eslint are used for the underlying @hey-api/openapi-ts, don't know why we would use biome over those tools.

Reply:
I see. I just ran it again and indeed confirmed that Prettier is working. This is fine as it is. 
However, I believe that the generated code should not be changed depending on whether Prettier is installed or not. Therefore, it might be a good idea to add comments like eslint-disable to the generated code, similar to what was done with the previous openapi-typescript-codegen.

Reply:
Agreed, adding that would be good in my opinion.